v2/thirdparty/common-customizations/sessions/share-session-across-sub-domains.mdx

```diff
@@ -59,11 +59,7 @@ SuperTokens.init({
 
```

Comment:
There are still code snippets for angular and vue wherein we use web-js for pre built UI. See here: https://deploy-preview-579--admiring-bhabha-7b1be9.netlify.app/docs/emailpassword/common-customizations/sessions/protecting-frontend-routes